### PR TITLE
RD traitors can now purchase an Omnilathe board

### PR DESCRIPTION
**Pull request by ghost**
:cl: optional name here
add: Traitor Research Directors can now purchase an Omnilathe board.
/:cl:

[why]:
Omnilathe board is just a standard, non-departmental locked protolathe board.
High-cost item for printing any research items. It's balanced by not being stealthy, as you have to build the machine somewhere, and actually steal materials required for building the items. Could be made 20 TC cost if wanted.

### CODE REVIEW

**Comment by kevinz000** [comment]
Please make the PR announce with the proper title.

**Comment by kevinz000** [comment]
Also why are you coding something I said as a joke?
This is not going to get you valided. No sane person will do that when they can just print gear.from it instead.

**Comment by kevinz000** [comment]
Also steal what materials they're the fucking Rd there's nothing to steal.

**Comment by awesine** [comment]
because you said it as a joke and I thought it was a good idea

>Also why are you coding something I said as a joke?

**Comment by optimumtact** [comment]
Please consider this a friendly warning, your PR's must have a properly descriptive title and body content that explains the purpose of the PR.

Your current title and or body content does not meet requirements, so please update it.

**Comment by awesine** [comment]
>Please make the PR announce with the proper title.


changed title, i forgot to change branch name

**Comment by lzimann** [comment]
By the way, this is the kudzu branch, which has been denied already.

**Comment by awesine** [comment]
wait what the fuck?

**Comment by awesine** [comment]
>By the way, this is the kudzu branch, which has been denied already.

i fucking made a new branched and changed the code, why is it kudzu?

**Comment by awesine** [comment]
let me try again

**Comment by awesine** [comment]
#34556
